sdks/python/apache_beam/runners/common.pxd

```diff
@@ -37,22 +37,20 @@ cdef class DoFnRunner(Receiver):
   cdef object tagged_receivers
   cdef LoggingContext logging_context
   cdef object step_name
-  cdef bint is_new_dofn
   cdef list args
   cdef dict kwargs
   cdef ScopedMetricsContainer scoped_metrics_container
   cdef list side_inputs
-  cdef bint has_windowed_side_inputs
+  cdef bint windowed_inputs
```

Comment:
Name this `has_windowed_inputs` maybe, because it still a boolean and not a list of inputs.